executable/te.ksy

```diff
@@ -0,0 +1,125 @@
+meta:
+  id: te
+  title: TE (Terse Executable) file format. A stripped version of PE (Portable Executable) format.
+  application: UEFI
+  file-extension:
+    - efi
+    - te
+  ks-version: 0.7
+  license: CC0-1.0
+  endian: le
+doc-ref: https://uefi.org/sites/default/files/resources/PI_Spec_1_6.pdf
```

Comment:
Can I ask you to add the description you already have in this PR as top-level `doc` element?